### ISSUE TITLE
Tailwind runs although it shouldn't

### ISSUE THREAD

**Issue by NiklasEi** [leptos-rs/cargo-leptos#254]
To my understanding of the readme, Tailwind should only run if `tailwind-input-file` is set. I do not have it set, but Tailwind still runs during `cargo leptos watch`.

**Comment by NiklasEi** [comment]
This currently fails my build, because I have multiple scss modules that get combined by [basro/stylance-rs](https://github.com/basro/stylance-rs) and are not all valid without the other modules due to CSS variables.

**Comment by nvim-ftw** [comment]
Dug into some source code. Why is there tailwind stuff in the `compile_sass` function?
```rust
// ...
pub async fn compile_sass(style_file: &SourcedSiteFile, optimise: bool) -> Result<Outcome<String>> {
    let mut args = vec![style_file.source.as_str()];
    optimise.then(|| args.push("--no-source-map"));

    let exe = Exe::Sass.get().await.dot()?;

    let mut cmd = Command::new(exe);
    cmd.args(&args);

    log::trace!(
        "Style running {}",
        GRAY.paint(format!("sass {}", args.join(" ")))
    );

    match wait_piped_interruptible("Tailwind", cmd, Interrupt::subscribe_any()).await? {
        CommandResult::Success(output) => Ok(Outcome::Success(output.stdout())),
        CommandResult::Interrupted => Ok(Outcome::Stopped),
        CommandResult::Failure(output) => {
            log::warn!("Tailwind failed with:");
            println!("{}", output.stderr());
            Ok(Outcome::Failed)
        }
    }
}
```

**Comment by benwis** [comment]
To me that looks like someone copy pasted the tailwind one and forgot/didn't care to change the inner tailwind naming

**Comment by nvim-ftw** [comment]
@benwis Sounds likely to me. If I find the root of this issue and get around to opening a PR, I'll include a fix for that. It doesn't seem urgent, just problematic for debugging. @NiklasEi did you think Tailwind was failing because it said "Tailwind" (understandable) or did you confirm that? Might not be tailwind.

**Comment by nvim-ftw** [comment]
Also, I'll open a separate issue for whatever is going on right now with mine, since it seems not to be quite right for this issue.

**Comment by NiklasEi** [comment]
> @benwis Sounds likely to me. If I find the root of this issue and get around to opening a PR, I'll include a fix for that. It doesn't seem urgent, just problematic for debugging. @NiklasEi did you think Tailwind was failing because it said "Tailwind" (understandable) or did you confirm that? Might not be tailwind.

I did not confirm it by looking into the code. It might not have been Tailwind in the end then, just Sass.

**Comment by benwis** [comment]
@nvim-ftw Would you like to open a quick PR to patch it to say Sass and not Tailwind, so more people are not confused?

**Comment by nvim-ftw** [comment]
I'll do it tonight!

**Comment by NiklasEi** [comment]
Resolved in #267
